effectiveOfficeBackend/src/main/kotlin/office/effective/feature/auth/converters/UserModelEntityConverter.kt

```diff
@@ -0,0 +1,27 @@
+package office.effective.feature.auth.converters
+
+import office.effective.feature.auth.repository.UserEntity
+import office.effective.model.UserModel
+
+class UserModelEntityConverter {
+    fun modelToEntity(userModel: UserModel): UserEntity {
+        var res = UserEntity()
+        res.id = userModel.id!!
```

Comment:
Try to get rid of "!!". You should use such an operator only if you are 100% sure that you won't have null here, but the compiler doesn't have enough information to realize that. I'm not sure if this is your case